pkg/ncdmv/types.go

```diff
@@ -88,20 +88,108 @@ type Location int
 
 const (
 	// Values taken from the "data-id" attribute set on each location div.
-	LocationInvalid      Location = iota
-	LocationAhoskie      Location = 99
-	LocationCarrboro     Location = 140
-	LocationCary         Location = 66
-	LocationClayton      Location = 42
-	LocationDurhamEast   Location = 47
-	LocationDurhamSouth  Location = 80
-	LocationFuquayVarina Location = 38
-	LocationGarner       Location = 69
-	LocationHillsborough Location = 52
-	LocationRaleighEast  Location = 181
-	LocationRaleighNorth Location = 10
-	LocationRaleighWest  Location = 9
-	LocationWendell      Location = 39
+	LocationAberdeen          Location = 100
+	LocationAhoskie           Location = 99
+	LocationAlbemarle         Location = 87
+	LocationAndrews           Location = 142
+	LocationAsheboro          Location = 97
+	LocationAsheville         Location = 124
+	LocationBoone             Location = 125
+	LocationBrevard           Location = 101
+	LocationBrysonCity        Location = 126
+	LocationBurgaw            Location = 111
+	LocationBurnsville        Location = 68
+	LocationCarrboro          Location = 140
+	LocationCary              Location = 66
+	LocationCharlotteEast     Location = 120
+	LocationCharlotteNorth    Location = 175
+	LocationCharlotteSouth    Location = 86
+	LocationCharlotteWest     Location = 121
+	LocationClayton           Location = 42
+	LocationClinton           Location = 112
+	LocationClyde             Location = 102
+	LocationConcord           Location = 141
+	LocationDurhamEast        Location = 47
+	LocationDurhamSouth       Location = 80
+	LocationElizabethCity     Location = 65
+	LocationElizabethtown     Location = 79
+	LocationElkin             Location = 103
+	LocationErwin             Location = 95
+	LocationFayettevilleSouth Location = 118
+	LocationFayettevilleWest  Location = 119
+	LocationForestCity        Location = 57
+	LocationFranklin          Location = 139
+	LocationFuquayVarina      Location = 38
+	LocationGarner            Location = 69
+	LocationGastonia          Location = 59
+	LocationGoldsboro         Location = 40
+	LocationGraham            Location = 137
+	LocationGreensboroEast    Location = 138
+	LocationGreensboroWest    Location = 143
+	LocationGreenville        Location = 22
+	LocationHamlet            Location = 104
+	LocationHavelock          Location = 82
+	LocationHenderson         Location = 70
+	LocationHendersonville    Location = 130
+	LocationHickory           Location = 78
+	LocationHighPoint         Location = 129
+	LocationHillsborough      Location = 52
+	LocationHudson            Location = 131
+	LocationHuntersville      Location = 19
+	LocationInvalid           Location = iota
```

Comment:
Please move `LocationInvalid` to the top.

Reply:
Done, happy to help. Let me know if you'd like anything else adjusted.